### DISCUSSION TITLE
File transfer time calculator

### DISCUSSION THREAD

**Discussion by revilo951** [CorentinTh/it-tools#833] [Ideas]
A handy tool would be one with two inputs - file size, and transfer speed (with perhaps dropdowns for size/speed units), and it tells you the time it would take to transfer the entered file size at the entered speed.

An online one I used to use a lot seems to have gone away.

**Comment by sharevb** [comment]
Hi @revilo951, #1289  and available here : https://sharevb-it-tools.vercel.app/data-transfer-rate-converter and if you arr interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)

**Reply by revilo951** [reply]
Nice work 👍 
Thanks
